Codecademy/client-modules
# fix(global header): add support for user role "account manager"

### Overview

Heard everyone wanted to see another global header PR!!!!!!!!!!!!

paired with @antoniablair to add this fix

Turns out we forgot to add support for the user role account manager in the global header.
An account manager is someone who can manage a business plan (ex. add or remove seats).
They can be either a pro or free user themselves. 






**Testing Instructions**

Visit [Preview link](https://6041b980ec56657648297dce--gamut-preview.netlify.app/storybook/?path=/docs/labs-experimental-molecules-globalheader--business-account-manager-pro)
Go to Global Header story
View the Business Account Manager pro & free stories. 
Both user types should see the "Business Account Management" link in the profile dropdown when on the desktop version of the header.
In mobile, neither should see the link as the page isn't ready to be viewed on mobile according to the business team engineers!

### PR Checklist

- [x] Related to JIRA ticket: [EGG-587](https://codecademy.atlassian.net/jira/software/projects/EGG/boards/142?selectedIssue=EGG-857)
